### ISSUE TITLE
--output-format=parseable and --output-format=msvs not working with pylint.epylint

### ISSUE THREAD

**Issue by DudeNr33** [PyCQA/pylint#4282]
It seems like pylint.epylint does not correctly support all output formats.
`--output-format=json` works fine, `--output-format=parseable` does not.

### Steps to reproduce

Given a file `a.py`:
```python
def main():
    print("Hello World")

```

### Current behavior

Result of `pylint a.py --output-format=parseable`:
```
************* Module a
testfile.py:1: [C0114(missing-module-docstring), ] Missing module docstring
testfile.py:1: [C0116(missing-function-docstring), main] Missing function or method docstring

------------------------------------------------------------------
Your code has been rated at 0.00/10 (previous run: 0.00/10, +0.00)
```

Result of `python -m pylint.epylint a.py --output-format=parseable`:
```
************* Module a
testfile.py:1: convention (C0114, missing-module-docstring, ) Missing module docstring
testfile.py:1: convention (C0116, missing-function-docstring, main) Missing function or method docstring

------------------------------------------------------------------
Your code has been rated at 0.00/10 (previous run: 0.00/10, +0.00)
```

### Expected behavior
`epylint` should either fully support parseable output or inform the user about an invalid choice by raising `InvalidReporterError` (the former would be my preferred solution).

Note that `--output-format=json` works as expected, while `--output-format=msvs` seems to have the same issue.

### pylint --version output

Result of `pylint --version` output:
```
pylint 2.7.4
astroid 2.5.2
Python 3.8.2 (default, Dec 21 2020, 15:06:03) 
[Clang 12.0.0 (clang-1200.0.32.29)]
```

**Comment by DudeNr33** [comment]
It seems like the problem is caused by the fact that `epylint.lint` explicitly defines the `--msg-template` option which takes precedence over `--output-format`.

``` Python
run_cmd = "import sys; from pylint.lint import Run; Run(sys.argv[1:])"
cmd = (
    [sys.executable, "-c", run_cmd]
    + [
        "--msg-template",
        "{path}:{line}: {category} ({msg_id}, {symbol}, {obj}) {msg}",
        "-r",
        "n",
        child_path,
    ]
    + list(options)
)
process = Popen(
    cmd, stdout=PIPE, cwd=parent_path, env=_get_env(), universal_newlines=True
)
```

That is to say, it takes precedence as long as the output format is not json.
It thus seems to be less of a bug and more of something that could some more documentation, either in the [output section](http://pylint.pycqa.org/en/latest/user_guide/output.html?highlight=msg-template#pylint-output) or the [section giving the example on how to use epylint](http://pylint.pycqa.org/en/latest/user_guide/run.html?highlight=epylint#invoking-pylint).
